lib/Controller/SignalingController.php

```diff
@@ -325,11 +326,13 @@ private function backendAuth($auth) {
 		return new JSONResponse($response);
 	}
 
-	private function backendRoom($room) {
-		$roomId = $room['roomid'];
-		$userId = $room['userid'];
+	private function backendRoom($roomRequest) {
+		$roomId = $roomRequest['roomid'];
+		$userId = $roomRequest['userid'];
+		$sessionId = $roomRequest['sessionid'];
+
 		try {
-			$room = $this->manager->getRoomForParticipantByToken($roomId, $userId);
+			$room = $this->manager->getRoomByToken($roomId);
```

Comment:
I think the only thing that needed a change was to make sure user id is `null` instead of `empty string` for guests?